src/chunkserver/datastore/chunkserver_chunkfile.h

```diff
@@ -364,7 +369,7 @@ class CSChunkFile {
     }
 
     inline int SyncData() {
-        return lfs_->Sync(fd_);
+        return lfs_->Fdatasync(fd_);
```

Comment:
Has this function ever been called? Generally, specify the type of flash disk when Open?

Reply:
use sync  in the background